Miniforge3/construct.yaml

```
@@ -37,7 +37,7 @@ user_requested_specs:
 specs:
   - python 3.12.*
   - conda {{ version.split("-")[0] }}
-  - mamba {{ mamba_version }}
+  - mamba >={{ mamba_version }}
```

```suggestion
  - mamba {{ mamba_version }}
```

This needs to be `==` (omitted) for reproducibility.